clients/cli/src/main/java/org/apache/gravitino/cli/commands/CreateTable.java

```diff
@@ -82,27 +82,47 @@ public void handle() {
       client = buildClient(metalake);
     } catch (NoSuchMetalakeException err) {
       exitWithError(ErrorMessages.UNKNOWN_METALAKE);
+      return; // Stop execution
     } catch (Exception exp) {
       exitWithError("Error initializing client or table name: " + exp.getMessage());
+      return;
```

Comment:
Please remove the unneeded return statements. This is inconsistent with other files here.